<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderTempo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserSets">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>271</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelT">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValorTempo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderMaximo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderMinino">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMinino">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMaximo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinimo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaximo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>82</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDisconnect">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>82</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>----------------</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIps">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderTempo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelValorTempo</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMinino</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMinino</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStart_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tcpConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMaximo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMaximo</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDisconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tcpDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stopData()</slot>
  <slot>tcpConnect()</slot>
  <slot>tcpDisconnect()</slot>
 </slots>
</ui>
